tests/integration/thrift_tests.py

```diff
@@ -455,7 +462,7 @@ def example(self, context):
                     svc.example()
 
         # this should be 1 second (1000 ms)
-        assert handler.context.headers.get(b"Deadline-Budget").decode() == str(1000)
+        assert handler.context.deadline_budget == 1.0  # pool timeout seconds
```

Comment:
In general using `==` on floats will cause issues and we usually use something like https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual instead. Same for other `==` asserts.